components/webfield/ProgramChairConsole/PaperStatus.js

```diff
@@ -143,13 +276,25 @@ const PaperStatus = ({ pcConsoleData, loadReviewMetaReviewData }) => {
     } else {
       const { notes, noteNumberReviewMetaReviewMap } = pcConsoleData
       if (!notes) return
-      const tableRows = [...(noteNumberReviewMetaReviewMap.values() ?? [])]
+      const actualNotes = noteContentField
+        ? notes.filter((note) => Object.keys(note.content).includes(noteContentField.field))
+        : notes
+      const actualNoteNumbers = actualNotes.map((note) => note.number)
+      const actualNoteNumberReviewMetaReviewMap = noteContentField
+        ? new Map(
+            [...noteNumberReviewMetaReviewMap].filter(([noteNumber, dataMap]) =>
+              actualNoteNumbers.includes(noteNumber)
+            )
+          )
+        : noteNumberReviewMetaReviewMap
+
+      const tableRows = [...(actualNoteNumberReviewMetaReviewMap.values() ?? [])]
```

Comment:
Same here... can just do
```suggestion
      const tableRows = actualNoteNumberReviewMetaReviewMap.values()
```